packages/web/components/transactions/transaction-rows.tsx

```diff
@@ -10,11 +10,13 @@ import { useAmplitudeAnalytics, useTranslation } from "~/hooks";
 
 export const TransactionRows = ({
   transactions,
+  selectedTransaction,
```

Comment:
Similar to prev comment, thought on just using the hash as the ID for the selected tx? We access and compare the hash anyways, this makes it a bit easier to set the selected transaction: perhaps, in the scenario some piece of code only has the hash and not the whole object.